### PR TITLE
PHOENIX-6410 Expose primary keys metadata via SqlAlchemy

### PR DESCRIPTION
**Pull request by stoty**

### CODE REVIEW

**Comment by romainr** [comment]
I can see for pip installing your branch and testing within Hue

**Comment by romainr** [comment]
Nice!

image

File: python-phoenixdb/phoenixdb/tests/test_db.py [hunk 1]
```diff
@@ -172,6 +172,66 @@ def test_meta(self):
...
+                    self.assertTrue(False)
+                except ProgrammingError:
+                    pass
+
+                self.maxDiff = None
+
+                self.assertEqual(meta.get_index_info(table='NON_EXISTENT'), [])
+
+                self.assertTrue(len(meta.get_index_info(table='DEFAULT_TABLE')) > 1)
+                print(meta.get_index_info(table='DEFAULT_TABLE'))
```
Comment (romainr) [inline python-phoenixdb/phoenixdb/tests/test_db.py:233]: nit: print

**Review by romainr — APPROVED** [review]
Also confirming we can get the indexes

`[{'name': 'GLOBAL_IDX', 'unique': True, 'column_names': ['0:MESSAGE', ':ID']}]`

File: python-phoenixdb/phoenixdb/tests/test_db.py (hunk 1 above)
Comment (stoty) [reply python-phoenixdb/phoenixdb/tests/test_db.py]: THX, done.

**Comment by stoty** [comment]
I have also added some post-processing in the SQLAlchemy index column list to get the expected name when there is no column family trickery.

**Review by joshelser — COMMENTED** [review]
Belated +1 from me :)

**Pull request closed without merging**